ouroboros-consensus/src/ouroboros-consensus/Ouroboros/Consensus/Mempool/Impl/Common.hs

```diff
@@ -236,7 +236,7 @@ initMempoolEnv ledgerInterface cfg capacityOverride _tracer txSize = do
       , mpEnvStateVar         = isVar
       , mpEnvAddTxsRemoteFifo = addTxRemoteFifo
       , mpEnvAddTxsAllFifo    = addTxAllFifo
-      , mpEnvTracer           = nullTracer
+      , mpEnvTracer           = tracer
```

Comment:
I wanted to look into #215, but I can't easily test it in CI  if the tests fail early because of the missing mempool tracer. We don't have to merge this change into the feature branch, but it will unblock me for now